sdk/rust/src/http.rs

```diff
@@ -461,14 +461,24 @@ impl IncomingResponse {
     /// # Panics
     ///
     /// Panics if the body was already consumed.
-    pub fn into_body_stream(self) -> impl futures::Stream<Item = Result<Vec<u8>, streams::Error>> {
+    // TODO: This should ideally take ownership of `self` and be called `into_body_stream` (i.e. symmetric with
```

Comment:
Sigh... I think this whole API might need another look - it's hard to get the flow right. Oh well...